nodes/widgets/ui_slider.html

```diff
@@ -145,6 +148,23 @@
                 </div>
             </div>
         </div>
+        <div class="form-row">
+            <label for="node-input-color"><i class="fa fa-adjust"></i> Color</label>
+            <div style="display: grid; grid-template-columns: repeat(3, 1fr); width: 70%">
+                <div>
+                    <span for="node-input-color">main</span>
```

Comment:
```suggestion
                    <span for="node-input-color">slider</span>
```

Think this reads a little better than "main"